visiont/models.py

```diff
@@ -26,22 +26,22 @@ def __init__(self, num_classes, C=3, H=224, W=224, P=16):
         self.class_token = nn.Parameter(torch.rand(D))
 
         # Standard transformer encoder, baseline defaults
-        self.enc = nn.TransformerEncoder(num_layers=12,
-                encoder_layer=nn.TransformerEncoderLayer(D, nhead=12, dim_feedforward=3072, activation="gelu"))
+        self.enc = nn.TransformerEncoder(
+            num_layers=6,
```

Comment:
Why the change to 6 here? Just to experiment?

Reply:
Same with heads below

Reply:
And feedforward

Reply:
yeah iterate faster for now. Else its slow and batch size is small